pom.xml

```diff
@@ -22,37 +22,43 @@
     <akka-management.version>1.0.7</akka-management.version>
     <akka-http.version>10.1.12</akka-http.version>
     <scala.compat.version>2.13</scala.compat.version>
```

Comment:
do we use that property name `scala.compat.version` in many places? somewhat strange name in my opinion.

Reply:
i'll switch that to just scala version

Reply:
I think we usually use `scala.binary.version` nowadays

Reply:
👍